spec/controllers/users_controller_spec.rb

```diff
@@ -3,13 +3,89 @@
 RSpec.describe UsersController, type: :controller do
 
   context 'GET #new' do
-    it 'should success' do
+    it 'should success render new' do
       get :new
       expect(response).to have_http_status(200)
+      expect(assigns(:user)).to be_a(User)
+      expect(assigns(:user)).to be_a_new(User)
```

Comment:
`be_a_new`, please explain why to have this ?

Reply:
it was another way of testing, different from the line above